types/discord.ts

```diff
@@ -17,7 +17,6 @@ import {
   GuildNsfwLevel,
   IntegrationExpireBehaviors,
   InteractionTypes,
-  Locales,
```

Comment:
intentional?

Reply:
yes. it's unused so it's removed. should i keep that unrelated changes or not

Reply:
actually i just merge main into this branch and it's already removed?